core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts

```diff
@@ -137,7 +137,7 @@ export default {
   } as const,
 
   ViewportWidthBreakpoints: {
-    MOBILE_PX: 768,
+    MOBILE_PX: 769,
```

Comment:
Looking at the PR, I don't think this is intended. If it is, why are we changing this?

Reply:
It's changed because it causes it to switch to mobile mode when it is LESS than 768 and not less than or equal to. Even before the changes I made it's still slightly incorrect.